### DISCUSSION TITLE
useForm({mode: 'all'}) with Controller component doesn't work properly

### DISCUSSION THREAD

**Discussion by VictorBoca** [react-hook-form/react-hook-form#2408] [Q&A] [answered]
Hi

I have a form with mode all and I have there components that user register and a component that use Controller. All the components that user register are working as expected onBlur and onChange they are giving the proper errors, but the Controller component it doesn't work as expected.

    <div>
        <textarea
          name={'textarea'}
          placeholder={'Text here'}
          ref={register({ required: { value: true, message: 'message' } })}
        />

        <Controller
          control={control}
          name={'currencyCode'}
          render={({ onChange, onBlur, value }) => <input onChange={onChange} onBlur={onBlur} value={value} />}
          rules={{ required: true }}
        />
      </div>

When i use mode: 'onBlur' is working as expected

**Comment by bluebill1049** [comment, marked as answer]
there is a bug on it already, fix is coming.

**Comment by bluebill1049** [comment]
https://github.com/react-hook-form/react-hook-form/issues/2406

**Comment by VictorBoca** [comment]
Thanks
